KtSPR--L4-T4--тренировки

DenisGubernatorov opened the pull request:
Вычисление упражнений для выполнения, в зависимости от дня с начала тренировок

DenisGubernatorov committed:
KtSPR--L4-T4--тренировки
Вычисление упражнений для выполнения, в зависимости от дня с начала тренировок

ievetrovdev commented on src/main/kotlin/lesson4/lesson4_task4.kt:
```diff
@@ -0,0 +1,27 @@
+
+    var trainingDay = 5
+    // Считаем как обычные люди, начинаем с 1. Таким образом смена упраженений происходит по четными дням тренировки
+    var isEvenTrainingDay = trainingDay.rem(2) == 0
```
есть более простая конструкция trainingDay % 2 == 0

ievetrovdev commented on src/main/kotlin/lesson4/lesson4_task4.kt:
```diff
@@ -0,0 +1,27 @@
+    var trainingDay = 5
+    // Считаем как обычные люди, начинаем с 1. Таким образом смена упраженений происходит по четными дням тренировки
+    var isEvenTrainingDay = trainingDay.rem(2) == 0
+    var handsEx = isEvenTrainingDay xor handsExFstDay
```
xor здесь и не нужен можно просто сравнивать через ==, и можно это делать сразу при выводе

DenisGubernatorov committed:
KtSPR--L4-T4--тренировки
1) Перевод логики вычисления какое упражнение выполнять с исключающего или на обычное сравнение
2) Вычисление перенесено из отдельных констант сразу в вывод
